src/styles-new/sass-base.scss

```diff
@@ -24,5 +24,6 @@
 // Meant for use as a single line import in Vue SFC's.
 // Do not include anything that renders to CSS!
 @import "constants";
-@import "constants-snow"; // TEMP
+@import "constants-espresso"; // TEMP
```

Comment:
@charlesh88 To confirm, this is the current "theme picker" line which we need to enable swapping of, right?

Reply:
Correct.